### ISSUE TITLE
vue create is broken with node version that doesn't respect  "^12.22.0 || ^14.17.0 || >=16.0.0".

### ISSUE THREAD

**Issue by nicotu01** [vuejs/vue-cli#6567]
### Version
4.5.12

### Environment info
```
Environment Info:

  System:
    OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
    CPU: (8) x64 Intel(R) Core(TM) i5-10310U CPU @ 1.70GHz
  Binaries:
    Node: 14.13.0 - ~/.nvm/versions/node/v14.13.0/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.13.0/bin/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v14.13.0/bin/npm
  Browsers:
    Chrome: 84.0.4147.89
    Firefox: 83.0
  npmGlobalPackages:
    @vue/cli: 4.5.8

```

### Steps to reproduce
vue create app

### What is expected?
succesfully create project

### What is actually happening?
error espree@8.0.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.13.0"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Found incompatible module.
 ERROR  command failed: yarn

**Comment by lemonKun** [comment]
vue 3.1.3 version broken，
use vue: ~3.0.0 in package.js temporarily

**Comment by sodatea** [comment]
Workaround:

Create a `.yarnrc` file in the project root, with the following content:
```
--install.ignore-engines true
```

Should be fixed in upstream: https://github.com/vuejs/vue-eslint-parser/issues/112

**Comment by sodatea** [comment]
Fixed in vue-eslint-parser@7.7.2

**Comment by nicotu01** [comment]
Thanks
